### ISSUE TITLE
Impossible to list or create terminal

### ISSUE THREAD

**Issue by niou128** [microsoft/vscode#133661]
Issue Type: <b>Bug</b>

Steps to Reproduce:

1. open terminal windows
2. try to create new terminal

The integrated terminal doesn't work. I have no functional terminal. I can't create new one. When I click on the plus button, nothing append. And I don't have any terminal profile in the dropdown list. 

![error1](https://user-images.githubusercontent.com/6471024/134480407-afbd4950-731c-4c89-98e5-e58a8a6d0ebd.png)

I added some profiles in my settings file, but it seems they are not recognized.

```
{
    "workbench.colorTheme": "Default Dark+",
    "terminal.integrated.profiles.windows": {
        "PowerShell": {
            "source": "PowerShell",
            "icon": "terminal-powershell"
        },
        "Command Prompt": {
            "path": [
                "${env:windir}\\Sysnative\\cmd.exe",
                "${env:windir}\\System32\\cmd.exe"
            ],
            "args": [],
            "icon": "terminal-cmd"
        },
        "Git Bash": {
            "source": "Git Bash"
        }
    },
    "terminal.integrated.defaultProfile.windows": "Command Prompt",
}
```

I tried to uninstall vs code, remove all folders and reinstall vs code but I have the same issue.

VS Code version: Code 1.60.2 (7f6ab5485bbc008386c4386d08766667e155244e, 2021-09-22T12:00:31.514Z)
OS version: Windows_NT x64 10.0.16299
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2808)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.85GB (1.35GB free)|
|Process Argv|--crash-reporter-id 9282e367-b9ea-4eba-8e7c-27a70d841326|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
armview|ben|0.4.5
azure-pipelines|ms-|1.191.0
vscode-azureappservice|ms-|0.23.0
vscode-azurefunctions|ms-|1.5.1
vscode-azureresourcegroups|ms-|0.4.0
vscode-azurestorage|ms-|0.12.1
vscode-azurevirtualmachines|ms-|0.4.1
vscode-bicep|ms-|0.4.613
vscode-cosmosdb|ms-|0.18.1
vscode-cosmosdbgraph|ms-|0.1.1
vscode-docker|ms-|1.17.0
vscode-language-pack-fr|MS-|1.60.4
csharp|ms-|1.23.15
vscode-dotnet-runtime|ms-|1.2.0
azure-account|ms-|0.9.8
azurecli|ms-|0.5.0
vscode-node-azure-pack|ms-|0.2.1
azurerm-vscode-tools|msa|0.15.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlang:30333561
binariesv615:30325510
vsccppwtct:30364498
pythonvssor306:30344512
bridge0708:30335490
pygetstartedt2:30360495
dockerwalkthrucf:30370837
bridge0723:30353136
pythonrunftest32:30365366
pythonf5test824cf:30361778
javagetstartedc:30364665
pythonvspyt187cf:30365362
pydsgsc2:30361791
vssid140:30363603
vssur157:30367808

```

</details>

**Comment by meganrogge** [comment]
What happens if you comment out your terminal profile settings?

**Comment by niou128** [comment]
I commented my profile settings and got the same result. Then I tried to remove all lines and got the same result.

**Comment by meganrogge** [comment]
any errors in the dev tools console?

**Comment by niou128** [comment]
There are no errors. Just a warning but it doesn't seem to be related.

![image](https://user-images.githubusercontent.com/6471024/135282721-d65e8dad-cf48-4f24-90d5-6fba0f467b89.png)

**Comment by meganrogge** [comment]
can you set log level to trace and copy and share the logs?

**Comment by niou128** [comment]
I don't have trace level, so I chose All levels. 

![image](https://user-images.githubusercontent.com/6471024/135396913-6cb4f300-562e-4a36-b275-1d2377a18e50.png)

**Comment by meganrogge** [comment]
Ctrl + Shift + P -> set log level -> trace

**Comment by niou128** [comment]
I don't know exactly what happened, but I have set log at the trace level.
![image](https://user-images.githubusercontent.com/6471024/135811569-de2861ea-4a5c-484a-b1a5-edbe761387e0.png)

Then I show output and set journal(fenêtre)
![image](https://user-images.githubusercontent.com/6471024/135812831-25591fb0-ea44-4312-97f8-51e0c36f8b3b.png)

I came back to the terminal and saw a very long list of open and unnamed terminals. I killed all of them and the normal list appeared with the different choices of terminals

My problem is now solved.
